crates/sui/src/dwallet_commands.rs

```diff
@@ -403,437 +403,439 @@ impl SuiDWalletCommands {
-                    })
-                    .unwrap()
-                    .clone();
-
-                sleep(Duration::from_millis(500)).await;
-                let mut cursor = None;
-                let mut sign_output: Option<SignOutput>;
-                loop {
-                    let client = context.get_client().await?;
-                    let response = client
-                        .read_api()
-                        .get_owned_objects(
-                            context.active_address()?,
-                            Some(SuiObjectResponseQuery::new(
-                                Some(SuiObjectDataFilter::StructType(SignOutput::type_())),
-                                Some(SuiObjectDataOptions::bcs_lossless()),
-                            )),
-                            cursor,
-                            None,
-                        )
-                        .await?;
-
-                    sign_output = response.data.iter().find_map(|o| {
-                        let move_object = o.move_object_bcs().unwrap();
-                        let sign_output = SignOutput::from_bcs_bytes(move_object).unwrap();
-                        if sign_output.session_id.bytes == session_id {
-                            Some(sign_output)
-                        } else {
-                            None
-                        }
-                    });
-
-                    if sign_output.is_some() {
-                        break;
-                    } else if response.has_next_page {
-                        cursor = response.next_cursor;
-                    } else {
-                        cursor = None;
-                    }
-                }
-                let sign_output = sign_output.unwrap();
-
-                let is_valid = verify_signature(
-                    messages_vec,
-                    &hash,
-                    dkg_output.public_key.clone(),
-                    sign_output.signatures.clone(),
-                );
-
-                println!("is_valid: {}", is_valid);
-
-                let signatures = sign_output
-                    .signatures
-                    .iter()
-                    .map(|s| Base64::encode(s))
-                    .collect::<Vec<_>>();
-
-                SuiClientCommandResult::NewSignOutput(NewSignOutput {
-                    dwallet_id,
-                    sign_output_id: sign_output.id.object_id().clone(),
-                    signatures,
-                })
-            }
+            // SuiDWalletCommands::Sign {
+            //     messages,
+            //     gas,
+            //     gas_budget,
+            //     serialize_unsigned_transaction,
+            //     serialize_signed_transaction,
+            //     hash,
+            // } => {
+            //     let hash: signature_mpc::twopc_mpc_protocols::Hash = hash.into();
+            //
+            //     let DWalletSecretShare {
+            //         alias: _,
+            //         dkg_output,
```

Comment:
Omer & I agreed to comment out this code for now, as it never worked & adding support for it seems redundant